offix-offline/src/main/java/org/aerogear/offixOffline/InputTypeChecker.kt

```diff
@@ -0,0 +1,14 @@
+package org.aerogear.offixOffline
+
+/*
+Class to check whether the input parameter type of the receiver class of mutation matches to the Input<*> class of Apollo.
+ */
+class InputTypeChecker(string: String) {
+
+    val str = string
```

Comment:
Why this str variable?
Make the constructor parameter as `val` and you can use it directly in the inCheck() method